app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">FInalProject</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Hey.... can we remove this string? this was attached as part of default code... everything else is ok :)